core/contracts/financial-templates/implementation/SyntheticToken.sol

```diff
@@ -4,76 +4,73 @@ import "../../common/implementation/ExpandedERC20.sol";
 
 
 /**
- * @notice A burnable and mintable ERC20. The contract deployer will initially
- * be the only minter and burner as well as the owner who is capable of adding new roles.
- * The contract deployer will also be the only initial minter of the contract.
+ * @title Burnable and mintable ERC20.
+ * @dev The contract deployer will initially be the only minter and burner as well as the owner  who
+ * is capable of adding new roles.
  */
+
 contract SyntheticToken is ExpandedERC20, ERC20Detailed {
+    /**
+     * @notice Constructs the SyntheticToken.
+     * @param tokenName used to describe the new token.
```

Comment:
nit: prefer to not use the name of the parameter as the beginning of the descriptive sentence.

Reply:
I think the linter actually requires you to put the name of the parameter after the `@param` tag

Reply:
The format I recall is `@param variable-name description...`

Reply:
Sorry, let me clarify. I think the name should be there, but it shouldn't be considered part of the sentence.
Prefer:
```
@param tokenName the percentage of blah to blah
```
over
```
@param tokenName is the percentage of blah to blah
```

Reply:
@ptare pointed this out a while back, but the latter is confusing when rendered on the docs site.

Reply:
yup, thanks @mrice32 . I have refactored accordingly